tests/suites/test_suite_ssl.function

```diff
@@ -3720,6 +3732,19 @@ void tls13_read_early_data(int scenario)
     server_options.group_list = group_list;
     server_options.early_data = MBEDTLS_SSL_EARLY_DATA_ENABLED;
 
+#if defined(MBEDTLS_SSL_ALPN)
+    switch (scenario) {
+        case TEST_EARLY_DATA_SAME_ALPN:
+        case TEST_EARLY_DATA_DIFF_ALPN:
+        case TEST_EARLY_DATA_NO_LATER_ALPN:
+            client_options.alpn_list[0] = "ALPNExample";
+            client_options.alpn_list[1] = NULL;
+            server_options.alpn_list[0] = "ALPNExample";
+            server_options.alpn_list[1] = NULL;
+            break;
```

Comment:
Should we have tests with two entries in each of the `alpn_list`s?

Reply:
This would rather be a test of the determination of the ALPN during the handshakes than a test that we check well the ticket ALPN against the ALPN of the resumption handshake.